src/test/e2e/tests/specs.test.ts

```diff
@@ -135,14 +132,13 @@ describe('Finding a court or tribunal listing', () => {
     });
 
     it('should display 3 result', async() => {
```

Comment:
can we change should statement to 13 results as well

Reply:
Updated :)